### DISCUSSION TITLE
[FEATURE REQUEST] Variant information inclusion in generated variants when using seed fuzzing.

### DISCUSSION THREAD

**Discussion by Nesuhst** [hlky/stable-diffusion-webui#336] [Feature suggestions]
The inclusion of "variant_of_seed: [original seed used to create variant]" and "variant_amount: [number]" in the sample info files could be useful.

**Comment by alijaya** [comment]
Yes, I think the variant seed should be included in the metadata, so we could replicate the exact image.
